tenants/ai-example/single-model-serving-vllm/base/kustomization.yaml

```diff
@@ -6,6 +6,6 @@ namespace: ai-example-single-model-serving
 helmCharts:
 - name: vllm-kserve
   releaseName: granite
-  version: 0.3.7
+  version: 0.4.1
```

Comment:
0.4 uses a slightly different values file format so we should stick to 0.3.7 for now